### PR TITLE
[UPD] Switch to Github Actions

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This one is not going to be easy...

**Comment by rousseldenis** [comment]
> This one is not going to be easy...

Not so hard.

Deactivated license mandatory check

**Comment by rousseldenis** [comment]
@LoisRForgeFlow @JordiBForgeFlow purchase_order_approval_block is LGPL but based on purchase_exception which is AGPL. This is forbiddden. IMHO you should go to AGPL

**Comment by rousseldenis** [comment]
@pedrobaeza :green_apple:

**Comment by pedrobaeza** [comment]
Great, I think we should directly switch that modules to AGPL and don't deactivate license check, but you mark the rhythm.

**Comment by rousseldenis** [comment]
> Great, I think we should directly switch that modules to AGPL and don't deactivate license check, but you mark the rhythm.

I usually ping the authors to warn them as I'd prefer to have their feeling about that.

Even if you are right, this check should pass anyway

**Comment by rousseldenis** [comment]
Following this https://github.com/odoo/odoo/commit/49b143e1b7f12ee6cf4d14754904f3e9b64a1011

It seems we don't need anymore purchase_order_line_sequence module.

@damdam-s @BernatPForgeFlow

**Comment by rousseldenis** [comment]
@pedrobaeza What do we do here ? Let license and ignore or change license ?

**Comment by pedrobaeza** [comment]
I would switch the license, as the author are warned and can make any later move, but keep the check for possible new contributions.

**Comment by rousseldenis** [comment]
> I would switch the license, as the author are warned and can make any later move, but keep the check for possible new contributions.

Done

File: purchase_order_line_sequence/__manifest__.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
         "views/report_purchasequotation.xml",
     ],
     "post_init_hook": "post_init_hook",
-    "installable": True,
+    "installable": False,
```
Comment (matteoopenf) [inline purchase_order_line_sequence/__manifest__.py:27]: why this key became false?
Which is the problem?

Comment (matteoopenf) [reply purchase_order_line_sequence/__manifest__.py:27]: https://github.com/OCA/purchase-workflow/pull/1564

**Pull request merged**